kdf.c

```diff
@@ -109,8 +89,12 @@ void kdf_decryption_key(const char *username, const char *password, int iteratio
 		iterations = 1;
 
 	if (iterations == 1)
-		sha256_hash(user_lower, strlen(user_lower), password, strlen(password), hash);
+		fail_invalid_iteration_count();
 	else
 		pbkdf2_hash(user_lower, strlen(user_lower), password, strlen(password), iterations, hash);
 	mlock(hash, KDF_HASH_LEN);
 }
+
+void fail_invalid_iteration_count() {
+	die("Action required: increase the iterations count through another client that supports iteration number setting.");
```

Comment:
I would perhaps change the text to:

`die("Action required: Your current iteration count does not meet the minimum number of %d iterations . Increase the iteration count on another client that supports iteration number setting.", MINIMUM_ITERATIONS);`

Currently its not that explicit why the current iteration count is not enough.